framework/decode/openxr_replay_consumer_base.h

```diff
@@ -484,7 +484,7 @@ class OpenXrReplayConsumerBase : public OpenXrConsumer
 
     VulkanReplayConsumerBase* vulkan_replay_consumer_ = nullptr;
 
-    std::vector<XrEventDataBuffer> received_events_;
+    std::deque<XrEventDataBuffer> previously_received_unhandled_events_;
```

Comment:
`XrEvenDataBuffer` are large, and maybe shuffled.  Do we want to store a unique_ptr to them instead (or use std::list to ensure no shuffling occurs)?

Reply:
Using a list now based on your suggestion.